docs/content/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -875,4 +875,13 @@ spec:
           storage: 50Gi
 ```
 
+## Flink Kubernetes Operator
```

Comment:
Maybe it would be better to simply include this information in the `Introduction` section. If we put it to the bottom like this, nobody will ever find it